gqcp/src/Processing/Properties/vAP1roGElectricalResponseSolver.cpp

```diff
@@ -0,0 +1,242 @@
+// This file is part of GQCG-gqcp.
+// 
+// Copyright (C) 2017-2019  the GQCG developers
+// 
+// GQCG-gqcp is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// GQCG-gqcp is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with GQCG-gqcp.  If not, see <http://www.gnu.org/licenses/>.
+// 
+#include "Processing/Properties/vAP1roGElectricalResponseSolver.hpp"
+
+#include "Mathematical/Optimization/LinearEquation/LinearEquationEnvironment.hpp"
+#include "Mathematical/Optimization/LinearEquation/LinearEquationSolver.hpp"
+
+
+namespace GQCP {
+
+
+/*
+ *  CONSTRUCTORS
+ */
+
+/**
+ *  @param vap1rog          the optimal vAP1roG parameters
+ */
+vAP1roGElectricalResponseSolver::vAP1roGElectricalResponseSolver(const QCModel::vAP1roG& vap1rog) :
+    vap1rog (vap1rog)
+{}
+
+
+
```

Comment:
```suggestion


```
I think previously we only used two white lines, even between two different blocks of functions (like constructors and public methods). At least I know it's like that in the two operators I have already implemented. But if I'm wrong, feel free to ignore this suggestion.